pkg/html/html.go

```diff
@@ -7,21 +7,43 @@ import (
 	"fmt"
 	"html/template"
 	"net/url"
+	"path/filepath"
 	"reflect"
 	"strings"
 	texttemplate "text/template"
 	"time"
 
 	"github.com/google/syzkaller/dashboard/dashapi"
 	"github.com/google/syzkaller/pkg/vcs"
+	"google.golang.org/appengine"
```

Comment:
Won't there be runtime conflicts because we import `google.golang.org/appengine/v2` in other places?

Reply:
Internally both do [return os.Getenv("GAE_ENV") == "standard"].
But just to be safe, let's fix it. Thanks!